sale_stock_ux/models/sale_order_line.py

```diff
@@ -187,7 +187,8 @@ def _compute_qty_delivered(self):
         super()._compute_qty_delivered()
         bom_enable = 'bom_ids' in self.env['product.template']._fields
         if bom_enable:
-            for line in self.filtered(lambda l: self.env['mrp.bom']._bom_find(
+            for line in self.filtered(
+                lambda l: l.product_id and self.env['mrp.bom']._bom_find(
```

Comment:
@nicomacr now that we have max-line-heigh 119 you could pull all of this into one line